activate.sh

```diff
@@ -17,8 +17,8 @@ autoenv_init() {
 	# We do this in a subshell so we can cd/chdir
 	_files="`
 		_hadone=''
-		while [ "${PWD}" != "${_mountpoint}" ]; do
-			_file="${PWD}/${AUTOENV_ENV_FILENAME}"
+		while [ "$(pwd -P)" != "${_mountpoint}" ]; do
+			_file="$(pwd -P)/${AUTOENV_ENV_FILENAME}"
```

Comment:
This can get wrong because the .env file might depend on being in the symlinked directory and not in the phsyical directory